bears/vcs/actions/AddNewlineAction.py

```diff
@@ -0,0 +1,31 @@
+from coalib.misc.Shell import run_shell_command
+from coalib.results.result_actions.ResultAction import ResultAction
+
+
+class AddNewlineAction(ResultAction):
+
+    SUCCESS_MESSAGE = 'New Line added successfully.'
+
+    def is_applicable(self,
+                      result,
+                      original_file_dict,
+                      file_diff_dict,
+                      applied_actions=()):
+        new_message, _ = run_shell_command('git log -1 --pretty=%B')
```

Comment:
User can first apply `EditCommitMessageAction` and change the commit message, so to check if `AddNewlineAction` is applicable or not, commit message it extracted first.